py/_io/terminalwriter.py

```diff
@@ -24,9 +24,14 @@
 
 
 def _getdimensions():
-    import termios,fcntl,struct
-    call = fcntl.ioctl(1,termios.TIOCGWINSZ,"\000"*8)
-    height,width = struct.unpack( "hhhh", call ) [:2]
+    if colorama is None:
```

Comment:
this platform check is incorrect, as colorama might be installed on non-windows environments

Reply:
i just noted that the same code is down below
so this is basically bad code duplication (see the win32 object handling)

Reply:
```
colorama = None
if sys.platform == "win32":
    try:
        import colorama
```

colorama will be `None` in non-windows environments since the import will not be attempted.